content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -0,0 +1,296 @@
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+First, set up a values configuration file, called `values.yaml` here:
+
+{{< text yaml >}}
+gateways:
+  istio-ingressgateway:
+    # Enable gateway injection
+    injectionTemplate: gateway
+    # Set a name for the gateway
+    name: ingressgateway
+    labels:
+      # Set a unique label for the gateway. This is required to ensure Gateways
+      # can select this workload
+      istio: ingressgateway
+{{< /text >}}
+
+Then install using standard `helm` commands:
+
+{{< text bash >}}
+$ kubectl create namespace istio-ingress
+$ helm install istio-ingress manifests/charts/gateways/istio-ingress -n istio-ingress -f values.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Managing gateways
+
+The following describes how to manage gateways after installation. For more information on their usage, follow
+the [Ingress](/docs/tasks/traffic-management/ingress/) and [Egress](/docs/tasks/traffic-management/egress/) tasks.
+
+### Gateway selectors
+
+The labels chosen on the gateway pods are used by Gateway selectors, so it is important that your
+Gateway objects are configured to match these labels.
+
+For example, in the deployments above we set the `istio=ingressgateway` label on the pods.
+To apply a Gateway to these deployments, we would select the same label:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+spec:
+  selector:
+    istio: ingressgateway
+...
+{{< /text >}}
+
+### Gateway deployment topologies
+
+Depending on your mesh configuration and use cases, you may wish to deploy gateways in different ways.
+Below shows a few different patterns for gateway deployments.
+Note that multiple options can be utilized within the same cluster.
+
+#### Shared gateway
+
+In this model, a single centralized gateway is used by many applications, possibly across many namespaces.
+Gateway(s) in the `ingress` namespace delegate ownership of routes to application namespaces, but retain control over TLS configuration.
+
+{{< image width="50%" link="shared-gateway.svg" caption="Shared gateway" >}}
+
+This model works well when you have many applications you want to expose externally, as they are able to use shared infrastructure.
+It also works well in models that have the same domain or TLS certificates shared by many applications.
+
+#### Dedicated application gateway
+
+In this model, an application namespace has its own dedicated gateway installation.
+This allows giving full control and ownership to a single namespace.
+This level of isolation can be helpful for critical applications that have strict performance or security requirements.
+
+{{< image width="50%" link="user-gateway.svg" caption="Dedicated application gateway" >}}
+
+Unless there is another load balancer in front of Istio, this typically means that each application will have its own IP address,
+which may complicate DNS configurations.
```

Comment:
@howardjohn for this model, is there any difference in the 3 choices of gateway installation methods?  Are all 3 support this?  I feel the operator and helm had some limitation where something are not customizable?

Reply:
That is related to upgrades only. This is more about the API model which doesn't care about install method